src/panel.js

```diff
@@ -529,20 +529,50 @@ const buildAllowedSitesPanel = async(panelId) => {
 
 const removeSiteFromContainer = async () => {
   const activeHostname = await getActiveHostname();
-  await browser.runtime.sendMessage( {removeDomain: activeHostname} );
-  browser.tabs.reload();
-  window.close();
+  buildRemoveSitePanel(activeHostname);
 };
 
-const addSiteToContainer = async () => {
-  const activeHostname = await getActiveHostname();
+const addSiteToContainer = async (activeHostname) => {
+  if (!activeHostname) { throw new Error("Missing site name. Cannot add site."); }
   const fbcStorage = await browser.storage.local.get();
   fbcStorage.domainsAddedToFacebookContainer.push(activeHostname);
   await browser.storage.local.set({"domainsAddedToFacebookContainer": fbcStorage.domainsAddedToFacebookContainer});
   browser.tabs.reload();
   window.close();
 };
 
+const buildAddSitePanel = async (siteName) => {
+  if (!siteName) {
```

Comment:
This accounts for adding a site from the first panel, where this data element has not be defined.